python/pyspark/rdd.py

```diff
@@ -56,6 +56,20 @@
 __all__ = ["RDD"]
 
 
+class PythonEvalType(object):
+    """
+    Evaluation type of python rdd.
+
+    These values are internal to PySpark.
```

Comment:
And should match with enum values in `org.apache.spark.api.python.PythonEvalType`?

Reply:
Right. Let me add it.

Reply:
Added